### DISCUSSION TITLE
[Question]: From which version at least, the vllm supports to infrence and serve Qwen2.5-14B-Instruct model?

### DISCUSSION THREAD

**Discussion by zengqingfu1442** [QwenLM/Qwen3#962] [Q&A] [answered]
### Has this been raised before?

- [X] I have checked [the GitHub README](https://github.com/QwenLM/Qwen2.5).
- [X] I have checked [the Qwen documentation](https://qwen.readthedocs.io) and cannot find an answer there.
- [X] I have searched [the issues](https://github.com/QwenLM/Qwen2.5/issues?q=is%3Aissue) and there is not a similar one.
- [X] I confirm that this is not a bug report, a feature request, or a badcase.

### Description

Which version of vllm support to inference and serve Qwen2.5-14B-Instruct model？

**Comment by jklj077** [comment, marked as answer]
the latest version is recommended. tested with v0.6.1. qwen2 arch (Qwen1.5, Qwen2, Qwen2.5) is supported by vllm since v0.3.0.

**Comment by thiner** [comment]
If you want to use the tools calling feature of Qwen2.5, you need to use version `0.6.0` or above.
